src/main/java/net/greatstart/dto/DtoProject.java

```diff
@@ -14,4 +15,6 @@
     @Valid
     private DtoProjectDescription desc;
 
+    private List<DtoInvestment> dtoInvestments;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "dtoInvestments" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)